Formula/jsoncpp.rb

```diff
@@ -22,15 +22,14 @@ class Jsoncpp < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "e00ec8c187ad0787b392125e31b23d0c2b0dcbb4e9f66cfd34570ca813146ab9"
   end
 
-  depends_on "meson" => :build
-  depends_on "ninja" => :build
+  depends_on "cmake" => [:build, :test]
```

Comment:
The CMake build is [deprecated](https://github.com/open-source-parsers/jsoncpp/wiki/Building#building-and-testing-with-cmake), so we should try to find a different solution for `securefs`.

We should probably also leave a comment here pointing this out, since we don't want to accidentally change this in the future. See also #103386.

Reply:
See https://github.com/netheril96/securefs/commit/205d507cc95b3851b3f861a2423c1f2752ea04dd, it's apparently been noted and fixed. It now simply, unconditionally, uses pkg-config when outside of vcpkg mode.